api/mess/views.py

```diff
@@ -1,3 +1,4 @@
+from ast import Is
```

Comment:
This import is not used. Make sure to not have unused imports.

Reply:
removed